pseudo.txt

```diff
@@ -71,7 +71,8 @@
       - playerBlack: Public. The black player instance of playerFactory.
 
     Methods:
-      - getPlayer(): returns the  
+      - getPlayer(colors): 
+        Public. Returns playerRed, playerBlack, or both.
```

Comment:
Is there ever a need to return both players? If it just returns red or black we could keep the argument as a boolean?